portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,8 +35,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      String userInput = request.getParameter("user-input");
+      String userIdea = request.getParameter("user-input");
```

Comment:
Does this ever have empty user input (empty string?) or do we enforce on the client side that this won't ever be empty? 
I think it'd make sense for the client to ensure a non empty input and then just document here that "its on the client to ensure that user input is non empty so here it's treated as such"